azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -427,6 +431,16 @@ private void cleanOlderProjects() {
     }
   }
 
+  public void deleteDirectory(ProjectVersion pv) throws IOException {
```

Comment:
deleteDirectory seems to have side effects. Would making it static makes mocking it later more difficult?